### DISCUSSION TITLE
FreeTAKServer Questions

### DISCUSSION THREAD

**Discussion by DaveyDave123** [FreeTAKTeam/FreeTakServer#397] [FreeTAKServer Support] [answered]
Hi,
I have a couple of questions about the Android version of the FreeTAKServer (v1.6) I had hoped you would be able to answer for me please:
1). Is it possible to run the Android FreeTAKServer and ATAK on the same phone? My current attempts have failed to connect the two.
2). Does the Android version of FreeTAKServer support the REST/json API?

Thanks for any info,
Dave

**Comment by brothercorvo** [comment, marked as answer]
1. yes, you can, but your phone may probably fry. Also not sure of the use case for it.
2. yes, it supports the 1.5 API, not all the extensions

**Comment by DaveyDave123** [comment]
Hi,
Thanks for your response.

I have one further question on the topic of API keys that I had hoped you could give me some pointers on please.

Basically, I am attempting to the use server API interface to extract information about entities etc… and am using the info here:

https://freetakteam.github.io/FreeTAKServer-User-Docs/API/REST_APIDoc/

I note that I need to add a key to the message header. The documentation suggests that we can use the Server GUI to create one. However, looking at the GUI, I can’t see any reference to API keys. Obviously, you can create users and define a ‘token’ with or without ‘Certs’, but I’m not sure whether that is the same thing?

Also, the example key in the documentation:

{
  "Authorization": "Bearer ***@***.***@guesmeIfyouCan"
}

Seems to consist of a username ‘meg’ followed by a @, another term, another @, etc…. Is there a convention for the structure of the key ie. ***@***.***, or similar?

Alternatively, if it were possible to turn off the authorization that would work for my purpose.

Thanks again for any pointers,
Regards,
Dave
